operator/Makefile

```diff
@@ -115,16 +115,24 @@ generate: $(CONTROLLER_GEN) ## Generate controller and crd code
 manifests: $(CONTROLLER_GEN) ## Generate manifests e.g. CRD, RBAC etc.
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
 
-test: deps generate go-generate lint manifests ## Run tests
+test: deps generate go-generate lint lint-prom manifests ## Run tests
```

Comment:
I'm undecided about the linting mentioned in `test` but not the unit test of the prometheus rules. Should we also unit-test the alerts every time?

Reply:
I agree that's it's weird to depend on `lint-prom` and not to run `test-unit-prom`.

I also realized now that I forgot to add  `test-unit-prom` to run in github actions. Looking at the workflow file, it seems that it runs `go test` directly rather than running it through `make test`
https://github.com/grafana/loki/blob/95fefa6096543a874a0645f2f2d6b094c7c09f0e/.github/workflows/operator.yaml#L91-L93

Would you like me to change the make rule to
```makefile
test: test-unit-prom deps generate go-generate lint lint-prom manifests ## Run tests
test: $(GO_FILES)
	go test ./... -coverprofile cover.out
	$(MAKE) test-unit-prom
```
and change the github action run to
```yaml
run: make test
```

?

Reply:
That's interesting indeed. I think I'd rather do the CI-related work in a separate PR, though, and focus on one topic.

I think we can leave it as it is, for now.